.github/vale/Vocab/Backstage/accept.txt

```diff
@@ -1,46 +1,123 @@
+_event
```

Comment:
What's the purpose of these changes? In general no code symbols should be needed here because Vale doesn't check things wrapped in code blocks

Reply:
My local Vale is checking everything and I get thousands of errors